lib/init.coffee

```diff
@@ -10,4 +15,42 @@ module.exports =
         type: 'string'
 
   activate: ->
-    console.log 'activate linter-gjslint'
+    @subscriptions = new CompositeDisposable
+    @subscriptions.add atom.config.observe 'linter-gjslint.executablePath',
+      (executablePath) =>
+        @executablePath = executablePath
```

Comment:
I'm a bit confused by the example - shouldn't `require('atom-package-deps').install()` take an optional callback instead of a string? 
https://github.com/travs/atom-package-dependencies/blob/master/index.js#L6